lessons/10_conditions/index.html

```diff
@@ -0,0 +1,512 @@
+					<li class="fragment">
+					<pre><code data-trim>
+							switch(condition){
+								// same as if(condition === "value1"){}
+								case "value1":
+									// do something
+									break; // end the switch
+								case "valueN": ...
+								default: // else
+									// do something
+							}
+					</code></pre>
+					</li>
+					<li class="fragment">The <code>switch</code> expression is evaluated once</li>
+					<li class="fragment">The value of the expression is compared with the values of each <code>case</code></li>
+					<li class="fragment">If there is a match, the associated block of code is executed</li>
+				</ul>
+			</section>
+			<section>
+				<h3>Enough words, show me your code!</h3>
+				<div class="fragment">
+					<pre><code data-trim>
+						alert(getFavouritePet(prompt("Who is the best pet?")));
+						function getFavouritePet(pet){
+							switch(pet){
+								case "goat":
+									alert("I ❤ goats too");
+								case "cat":
+									return "🐱 are also fun!";
+								case "":
+									alert("stop cheating!");
+								default:
+									alert("I still ❤ goats more than " + pet);
+						}
+						alert("I am outside of switch case block");
+						}
+					</code></pre>
+				</div>
+			</section>
+			<section>
+				<h3>New challenge: greet a super hero with <code>switch</code> block</h3>
+				<ul>
+					<li class="fragment">Super hero has a real name and a hero name</li>
+					<li class="fragment">Check if we are on a mission</li>
+					<li class="fragment">Remember plan B</li>
+					<li class="fragment">
+					<pre><code data-trim>
+						let onMission = prompt("Are we on a mission?");
+						switch(onMission) {
+						case "" :
+							alert();
+							break;
+						default : ... ;
+						}
+					</code></pre>
+					</li>
+				</ul>
+			</section>
+			<section>
+				<h3>Here is my solution</h3>
+				<div class="fragment">
+					<pre><code data-trim>
+						let heroName = "Rocket Raccoon"
+						let realName = "Peter Quill's pet";
+						greetHero(aa, bb)
+						function greetHero(aa, bb) {
+						switch(prompt("Are we on a mission?")){
+						case "yes":
+							alert("Hey there, " + aa + " 👋");
+							break;
+						case "no":
+							alert("Hey there, " + bb + " 👋");
+							break;
+						default:
+							alert(aa + bb);
+						}}
+					</code></pre>
+				</div>
+			</section>
+			<section>
+				<h3>So what we know so far</h3>
+				<ul>
+					<li class="fragment"><pre><code data-trim>
+						let name = "Peter";
+						alert("Hi " + name + "!");</code></pre>
+					</li>
+					<li class="fragment"><pre><code data-trim>
+					function greetSpiderMan(onMission){
+						if(onMission==="yes"){
+							return "SpiderMan"
+						}else {
+							return "Peter"
+						}
+					}
+					alert("Hi " + greetSpiderMan("no"));
+						</code></pre>
+					</li>
+					<li class="fragment"><pre><code data-trim>
+					function greetSpiderMan(onMission){
+						switch(onMission){
+							case "yes": return "SpiderMan"; break;
```

Comment:
I think if you have return then you don't need break after that, since you already closed the execution of that function.